Sprint-1/exercises/count.js

```diff
@@ -2,5 +2,9 @@ let count = 0;
 
 count = count + 1;
 
+// on line three 1 is added to the count variable by using the assignment operator =
```

Comment:
We have a term to describe operation like `count = count + 1` in programming. Would you try feeding the code to ChatGPT to see how else the code can be described? You may learn some terminology or more concise way to describe code from time to time.